raiden/api/python.py

```diff
@@ -213,8 +213,11 @@ def deposit(self, token_address, partner_address, amount):
         token.approve(netcontract_address, amount)
 
         # Obtain the netting channel and fund it by depositing the amount
+        old_balance = channel.contract_balance
         netting_channel = self.raiden.chain.netting_channel(netcontract_address)
         netting_channel.deposit(amount)
+        # Also make sure to update the netting_channel.py contract balance
+        channel.our_state.update_contract_balance(old_balance + amount)
```

Comment:
Hmmm I see. Then this is not a proper solution and we would need to wait in the API call for the processing of the event.

Let me do the changes and tell me if it works for you during testing.